src/containers/AssignmentTexterContact.jsx

```diff
@@ -249,7 +255,6 @@ class AssignmentTexterContact extends React.Component {
 
   getAvailableInteractionSteps(questionResponses) {
```

Comment:
Still getting an error on this screen:
```
10:51:13 AM server.1              |  TypeError: Cannot read property '0' of undefined
10:51:13 AM server.1              |      at getTopMostParent (/Users/shaka/moveon/Spoke/src/lib/interaction-step-helpers.js:83:10)
10:51:13 AM server.1              |      at _callee7$ (/Users/shaka/moveon/Spoke/src/server/api/campaign-contact.js:168:14)
10:51:13 AM server.1              |      at tryCatch (/Users/shaka/moveon/Spoke/node_modules/regenerator-runtime/runtime.js:65:40)
10:51:13 AM server.1              |      at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/shaka/moveon/Spoke/node_modules/regenerator-runtime/runtime.js:303:22)
10:51:13 AM server.1              |      at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/shaka/moveon/Spoke/node_modules/regenerator-runtim
10:51:13 AM server.1              |  >  e/runtime.js:117:21)
10:51:13 AM server.1              |      at step (/Users/shaka/moveon/Spoke/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
10:51:13 AM server.1              |      at /Users/shaka/moveon/Spoke/node_modules/babel-runtime/helpers/asyncToGenerator.js:28:13
10:51:13 AM server.1              |      at run (/Users/shaka/moveon/Spoke/node_modules/core-js/modules/es6.promise.js:66:22)
10:51:13 AM server.1              |      at /Users/shaka/moveon/Spoke/node_modules/core-js/modules/es6.promise.js:79:30
10:51:13 AM server.1              |      at flush (/Users/shaka/moveon/Spoke/node_modules/core-js/modules/_microtask.js:18:9)
10:51:13 AM server.1              |      at _combinedTickCallback (internal/process/next_tick.js:67:7)
10:51:13 AM server.1              |      at process._tickDomainCallback (internal/process/next_tick.js:122:9)
```

Reply:
Which I think has to deal with interactionSteps still coming in as an empty array

Reply:
@shakalee14 I'll look into this more, but do you still get this from a fresh campaign or one that might have been created with the previous version?

Reply:
this is with a fresh campaign - when attempting to use dynamic assignment